### PR TITLE
1. Overlay fix

### PR DESCRIPTION
**Pull request by ShaneKoNaung**
- [x] refactored print_annotations_pil
- [x] combining overlapping bbox
overlapping_bbox

- [x] blur original text while using translation overlay
Screenshot 2024-06-14 165502

- [x] separating different bbox based on vertical difference to avoid these errors

Before
multiple_dialogue_box_error

After
Screenshot 2024-06-14 165735

## WIP 
- [x] **Unit Tests**

- [x] Text Warping Errors 
Before
text_warping_error
After

test_annotate_translation_empty

- [ ] Translated text in place of original text

This is in another PR.

### CODE REVIEW

**Comment by mattkanwisher** [comment]
btw remove "WIP :" when the branch is ready to review

File: Test_dialogue_overlay.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
+import unittest
```
Comment (mattkanwisher) [inline Test_dialogue_overlay.py]: can we have a directory for tests?

Comment (mattkanwisher) [reply Test_dialogue_overlay.py]: also move unit_test_data into the test directory

Comment (ShaneKoNaung) [reply Test_dialogue_overlay.py]: Done

**Pull request merged**